projects/components/src/form/form-select/form-select.component.html

```diff
@@ -13,6 +13,7 @@
                         {{ option.isTranslatable ? (option.display | translate) : option.display }}
                     </option>
                 </select>
+                <clr-icon *ngIf="showErrors" class="clr-validate-icon" shape="exclamation-circle"></clr-icon>
```

Comment:
Did you load this into VCD UI and test it out on any forms in there? It would be nice to get a screenshot of what this looks like in VCD UI

Reply:
@ryan-bradford She will probable have to be guided on how to do that

Reply:
Done and done, we just had a good call